src/apps/deskflow-gui/MainWindow.cpp

```diff
@@ -445,6 +461,11 @@ void MainWindow::openHelpUrl() const
   QDesktopServices::openUrl(QUrl(kUrlHelp));
 }
 
+void MainWindow::openGetNewVersionUrl() const
+{
+  QDesktopServices::openUrl(QStringLiteral("https://github.com/deskflow/deskflow/releases/latest"));
```

Comment:
URLs should be constants so we can keep the URLs consistent across the app. For consistency, we should use `deskflow.org` as we do for other URLs.

```suggestion
  QDesktopServices::openUrl(kUrlDownload);
```

This will produce the URL: `https://deskflow.org/download?source=gui`
```patch
diff --git a/src/lib/gui/constants.h b/src/lib/gui/constants.h
index d6add12a..5825dd37 100644
--- a/src/lib/gui/constants.h
+++ b/src/lib/gui/constants.h
@@ -18,6 +18,7 @@ const auto kOrgDomain = QStringLiteral("deskflow.org");
 const auto kUrlSourceQuery = "source=gui";
 const auto kUrlApp = QStringLiteral("https://deskflow.org");
 const auto kUrlHelp = QString("%1/help?%2").arg(kUrlApp, kUrlSourceQuery);
+const auto kUrlDownload = QString("%1/download?%2").arg(kUrlApp, kUrlSourceQuery);
 
 #if defined(Q_OS_LINUX)
 const auto kUrlGnomeTrayFix = "https://extensions.gnome.org/extension/615/appindicator-support/";
```

Reply:
i don't see any reason to even use that abstraction. We only use that link once and if you do go to 
deskflow.org/download?source=gui its the release page not release latest. The update will only show on new stable release and they will always be our latest tag.